lib/build.gradle

```diff
@@ -25,6 +25,8 @@ repositories {
 }
 
 dependencies {
+    implementation 'org.jetbrains:annotations:24.0.0'
```

Comment:
This helps in development but doesn't offer anything helpful at compile-time, at least out of the box.